.github/workflows/labeler.yaml

```diff
@@ -15,4 +15,17 @@ jobs:
     - uses: actions/labeler@v5.0.0-alpha.1
       with:
         dot: true
-        configuration-path: .github/labeler.yaml
\ No newline at end of file
+        configuration-path: .github/labeler.yaml
+  pr_branch_check:
+    runs-on: ubuntu-latest
+    if: github.repository == 'commaai/openpilot'
+    steps:
+    - uses: Vankka/pr-target-branch-action@v2
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        target: /^(?!master$).*/
```

Comment:
does this not work?
```suggestion
        target: master
```

Reply:
target is the name of the branches that you are not allowed to make a pr against. So in this case we want anything other than "master"